### PR TITLE
add license secret and mgmt config map for r33 

### PR DESCRIPTION
**Pull request by AlexFenlon**
### Proposed changes

add initial r33 commit adds license token secret name and mgmt config map.
add app protect 5.4

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by AlexFenlon** [comment]
All feedback on the [previous pr](https://github.com/nginxinc/kubernetes-ingress/pull/6888) has been addressed

File: deployments/deployment/nginx-plus-ingress.yaml [hunk 1]
```diff
@@ -90,6 +90,7 @@ spec:
         args:
           - -nginx-plus
           - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
+          - -mgmt-configmap=$(POD_NAMESPACE)/nginx-config-mgmt
```
Comment (vepatel) [inline deployments/deployment/nginx-plus-ingress.yaml:93]: needs to be added in `/daemonset/nginx-plus-ingress.yaml` as well

File: internal/configs/configmaps_test.go [hunk 2]
```diff
@@ -285,6 +285,171 @@ func TestParseConfigMapAccessLogDefault(t *testing.T) {
...
+			}
+			if warnings {
+				t.Fatal("Unexpected warnings")
+			}
+
+			if result.EnforceInitialReport == nil {
+				t.Fatalf("EnforceInitialReport: want %v, got nil", *test.want.EnforceInitialReport)
+			}
+			if *result.EnforceInitialReport != *test.want.EnforceInitialReport {
+				t.Fatalf("EnforceInitialReport: want %v, got %v", *test.want.EnforceInitialReport, *result.EnforceInitialReport)
```
Comment (jjngx) [inline internal/configs/configmaps_test.go:447]: same here

**Review by jjngx — APPROVED** [review]
🚀

**Review by vepatel — APPROVED** [review]

**Pull request merged**